File: app/src/main/java/com/moviepass/model/PerformanceInfo.java
package com.moviepass.model;

/**
 * Created by anubis on 6/10/17.
 */

public class PerformanceInfo {

    String externalMovieId;
    String format;
    int tribuneTheaterId;
    int screeningId;
    String dateTime;
    int performanceNumber;
    String sku;
    Double price;
    String auditorium;
    String performanceId;
    String sessionId;
    int normalizedMovieId;

    public PerformanceInfo() {
    }

    public String getExternalMovieId() { return externalMovieId; }

    public String getDateTime() { return dateTime; }

    public int getNormalizedMovieId() { return normalizedMovieId; }

    public int getScreeningId() { return screeningId; }

    public String getFormat() { return format; }

    public int getTribuneTheaterId() { return tribuneTheaterId; }

    public int getPerformanceNumber() { return  performanceNumber; }

    public String getSku() { return sku; }

    public Double getPrice() { return price; }

    public String getAuditorium() { return auditorium; }

    public String getPerformanceId() { return performanceId; }

    public String getSessionId() { return sessionId; }
}

File: app/src/main/java/com/mobile/model/PerformanceInfo.kt
package com.mobile.model

import android.os.Parcelable
import com.mobile.reservation.TicketType
import kotlinx.android.parcel.Parcelize

@Parcelize
class PerformanceInfo(
        var externalMovieId: String? = null,
        var format: String? = null,
        var tribuneTheaterId: Int = 0,
        var screeningId: Int = 0,
        var dateTime: String? = null,
        var performanceNumber: Int = 0,
        var sku: String? = null,
        var price: Double? = null,
        var auditorium: String? = null,
        var performanceId: String? = null,
        var sessionId: String? = null,
        var normalizedMovieId: Int = 0,
        var ticketType: TicketType? = null,
        var cinemaChainId: String? = null,
        var showtimeId: String? = null
) : Parcelable
